src/main/java/de/melanx/skyblockbuilder/EventListener.java

```diff
@@ -60,6 +60,9 @@ public void onRegisterCommands(RegisterCommandsEvent event) {
                 .then(InviteCommand.register())
                 .then(ListCommand.register())
                 .then(TeamCommand.register())
+                .then(LeaveCommand.register())
+                .then(HomeCommand.register())
+                .then(CreateCommand.register())
```

Comment:
Sorry, I'm a bit strange. Should be sorted alphabetically. 
```suggestion
                .then(CreateCommand.register())
                .then(HomeCommand.register())
                .then(InviteCommand.register())
                .then(LeaveCommand.register())
                .then(ListCommand.register())
                .then(TeamCommand.register())
```